framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -2198,4 +2200,46 @@ private boolean isRelevantHelper(TypeMirror tm) {
                 throw new BugInCF("isRelevantHelper(%s): Unexpected TypeKind %s", tm, tm.getKind());
         }
     }
+
+    /**
+     * Return the type of the default value of the given type. The default value is 0, false, or
+     * null.
+     *
+     * @param typeMirror a type
+     * @return the annotated type of {@code type}'s default value
+     */
+    // TODO: Cache results to avoid recomputation.
+    public AnnotatedTypeMirror getDefaultValueAnnotatedType(TypeMirror typeMirror) {
+        return getAnnotatedType(getDefaultValueTree(typeMirror));
+    }
+
+    /**
+     * Return a tree for the default value of the given type. The default value is 0, false, or
+     * null.
+     *
+     * @param typeMirror a type
+     * @return a tree for {@code type}'s default value
+     */
+    private LiteralTree getDefaultValueTree(TypeMirror typeMirror) {
```

Comment:
This should be in `TreeUtils` because it imports a class from outside the support complier API, (com.sun.tools.javac.code.TypeTag).